janus_server/src/aggregator.rs

```diff
@@ -1362,6 +1382,33 @@ fn error_handler<R: Reply>() -> impl Fn(Result<R, Error>) -> warp::reply::Respon
     }
 }
 
+/// Factory that produces a closure that will wrap a `warp::Filter`, measuring the time that it
+/// takes to run, and recording it in metrics.
+fn timing_wrapper<F, T>(
+    value_recorder: &ValueRecorder<f64>,
+    name: &'static str,
+) -> impl Fn(F) -> BoxedFilter<(T,)>
+where
+    F: Filter<Extract = (T,), Error = Rejection> + Clone + Send + Sync + 'static,
+    T: Reply,
+{
+    let bound_value_recorder = value_recorder.bind(&[KeyValue::new("endpoint", name)]);
+    move |filter| {
+        warp::any()
+            .map(std::time::Instant::now)
```

Comment:
Any reason not to `use std::time::Instant` at the top of the file to declutter this?